packages/clarity-decode/types/interaction.d.ts

```diff
@@ -3,13 +3,18 @@ import { PartialEvent } from "./core";
 
 export interface ClickData extends Interaction.ClickData {
     hashBeta: string;
+    checksum?: string;
 }
 
 export interface TimelineData extends Interaction.TimelineData {
     hashBeta: string;
 }
```

Comment:
Where are the decode change that consume the checksum field here? Where are the changes that send it?

Reply:
Simplified this. We now support change event.